contracts/signature/ERC1271StoredMock.sol

```diff
@@ -8,7 +8,7 @@ import './ERC1271Stored.sol';
 contract ERC1271StoredMock is ERC1271Stored {
   constructor (
     bytes32 hash,
-    bytes memory
+    bytes memory signature
```

Comment:
Why name the variable? It's unused and therefore will cause a compilation warning.

Reply:
Not having this variable named prevents typechain from generating the typed contracts.

![image](https://user-images.githubusercontent.com/7013086/119603892-b03dc080-be20-11eb-962b-cec78dea697f.png)